tests/test_pca.py

```diff
@@ -62,13 +64,31 @@
 )
 
 
+def dask_array_dir_chunks(
+    a: object, *, conv: Callable[[object], DaskArray], dir: Literal["horiz", "vert"]
+):
+    arr = conv(a)
+    chunks = (arr.chunksize[0], -1) if dir == "horiz" else (-1, arr.chunksize[1])
+    return arr.rechunk(chunks)
+
+
+def svdify(array_types: Iterable[ParameterSet]) -> Generator[ParameterSet, None, None]:
```

Comment:
I think the name could be better - this is preparing for svd, not actually doing it?